libs/shared-angular/src/lib/oral-attendance/oral-attendance.component.html

```
@@ -0,0 +1,155 @@
+              (click)="undo()">
+              <mat-icon>undo</mat-icon>
+            </button>
+            <button
+              mat-icon-button
+              type="button"
+              [attr.aria-pressed]="!swipingEnabled()"
+              [attr.aria-label]="swipingEnabled() ? 'Desativar gestos de deslizar' : 'Ativar gestos de deslizar'"
+              (click)="toggleSwiping()">
+              <mat-icon>{{ swipingEnabled() ? 'swipe' : 'do_not_touch' }}</mat-icon>
+            </button>
+            @if (redoPersonIds().length > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)